### PR TITLE
Viro Reconfigure

### PR DESCRIPTION
**Pull request by MrLostman**
## About The Pull Request
**Bee Infestation**
-symbiotic and honey producing have been separated
`resistance 6 and 8 respectively`
-bee spawn chance linked to stage speed
`prob(25 + clamp(A.properties["stage_rate"] * 5, -25, 70)`
-can stop it from making bees but removes royal jelly,and halfs the amount of honey
`resistance 12`
**Pituitary Disruption**
-no longer spawns meta grudges
-no longer makes you vomit all over the station
-no longer randomises organs
**Nanite Symbiosis**
-no longer causes glitches
**Self Resp**
-farting removal modularized
**Viral Aggressive Meta**
-If it cures the virus does not make them immune
**Weight Loss**
-threshold to make it stop you from becoming fat
`resistance 8`
**Spiked Skin**
-returned to 60 armour cap
**Deoxyribonucleic Acid Saboteur**
-removed bad mutations from good mutation threshold
-removed stealth threshold
**Superficial Healing**
-healing threshold 15 - > 25, 30 -> 50

### Stat Changes
stealth, resistance, stage speed, trans
**Deafness**
-1,-2,-1,-3 -> 0, -2, -1, -1
**Fever**
-1,3,3,2 -> 0,3,3,2
**Spontaneous Combustion**
1,-2,-3,-3 -> 1,-1,-2,-1
**Necrotizing Fasciitis**
-3,-3,0,-4 -> -1,-1,0,-1
**Narcolepsy**
1,-2,-3,-2 -> 1,-1,-2,-1
**Hyphema**
-1,-3,-4,-2 -> -1,-2,1,-1
**Vomiting**
-2,1,0,2 -> -3,2,1,3
**Deoxyribonucleic Acid Saboteur**
-2,-3,0,-3 -> -1,-2,0,-2

## Changelog
:cl:
balance: rebalanced stats of deafness, fever, spontaneous combustion, necrotizing fasciitis, narcolepsy, hyphema and vomiting, DAS
refactor: modularized self resp fart removal
tweak: bee infestation symbiotic and honey producing have been separated, bee spawn chance linked to stage speed and a no bee threshold
tweak: pituitary disruption, superficial healing and nanite symbiosi has had zesko grief code removed
tweak: viral aggressive meta doesn't make host immune if it cures
tweak: weight loss threshold that stops people from getting fat
tweak: spiked skin armour reverted to 60
tweak: das stealth threshold removed and bad mutations removed from good threshold
tweak: superficial healing threshold increased to 25 and 50, zesk grief code removed
/:cl:

### CODE REVIEW

**Comment by TheFakeElon** [comment]
Spontaneous combustion buffed 😔

**Comment by MrLostman** [comment]
your thinking of the wrong one doris, 
image this is the one people use

**Comment by zeskorion** [comment]
"buffs nontot viro"
bro nontot viro is fine. you're just adding dumb powergame shit. Especially spiked skin.

**Comment by MrLostman** [comment]
The objective opinion of the person who made all the grief code compaired to all the people who say non tot viro has little to work with.

**Comment by zeskorion** [comment]
> The objective opinion of the person who made all the grief code compaired to all the people who say non tot viro has little to work with.

those people largely refer the the overabundance of tot shit in comparison, as well as the fact that these buffoons want to *spread* their *viruses*

**Comment by MCterra10** [comment]
@zeskorion stop being a dumbass and buffing your favorite job and we wouldn't need these PRs. Sadly bee is too thin-skinned to tell you "no"

**Comment by zeskorion** [comment]
> @zeskorion stop being a dumbass and buffing your favorite job and we wouldn't need these PRs. Sadly bee is too thin-skinned to tell you "no"

Stop buffing my favorite job so you... dont have to buff my favorite job? What?

I think you mean stop carefully nerfing my favorite job to keep it balanced

File: austation/code/datums/diseases/advance/symptoms/beesymptom.dm [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+				to_chat(M, "<span class='notice'>You can't get the taste of honey out of your mouth!.</span>")
+				if(no_bee)
+					M.reagents.add_reagent(/datum/reagent/consumable/honey,max(0, 5 - existing_honey))
+					M.reagents.add_reagent(/datum/reagent/medicine/insulin,max(0, 2.5 - existing_insulin))
+				else
+					M.reagents.add_reagent(/datum/reagent/consumable/honey,max(0, 10 - existing_honey))
+					M.reagents.add_reagent(/datum/reagent/consumable/honey/special,max(0, 5 - existing_royaljelly))
+					M.reagents.add_reagent(/datum/reagent/medicine/insulin,max(0, 5 - existing_insulin))
+
+			if(prob(25 + clamp(A.properties["stage_rate"] * 5, -25, 70)) && !no_bee)
```
Comment (ike709) [inline austation/code/datums/diseases/advance/symptoms/beesymptom.dm:86]: lol

**Comment by zeskorion** [comment]
> spite pr breaks shit
nice.

**Pull request merged**